tests/drivers/build_all/sensor/adc.dtsi

```diff
@@ -116,3 +116,13 @@ test_seeed_grove_temperature: seeed-grove-temperature {
 	compatible = "seeed,grove-temperature";
 	io-channels = <&test_adc 0>;
 };
+
+test_veaa_x_3: test_veaa_x_3 {
+	status = "okay";
+	compatible = "festo,veaa-x-3";
+	io-channels = <&test_adc 3>;
+	dac = <&test_adc>;
```

Comment:
`test_dac`?

Reply:
Oops!
I fixed it. Thank you.

Reply:
Hmm - I still see `test_adc` instead of `test_dac`. Could you revisit this?